pyproject.toml

```diff
@@ -124,5 +117,29 @@ plugins = ["pydantic.mypy"]
 [tool.pytest.ini_options]
 markers = [
     "integration: marks tests as integration (deselect with '-m \"not integration\"').",
-    "asyncio: marks tests as async."
+    "asyncio: marks tests as async.",
+]
+
+[tool.ruff]
+src = ["truss", "truss-chains", "truss-utils"]
+target-version = "py38"
+line-length = 88
+lint.extend-select = ["I"]
+lint.ignore = ["E402"]
+
+[tool.ruff.lint.isort]
+# Matches the Google Python Style Guide.
+section-order = [
+    "future",
+    "standard-library",
+    "third-party",
+    "first-party",
+    "local-folder",
 ]
+
+[tool.ruff.lint.extend-per-file-ignores]
+"**tests/samples.py" = ["I", "F"]
+"config.py" = ["I", "F"]
+
+[tool.ruff.lint.pycodestyle]
+max-line-length = 120
```

Comment:
This is 120, above there is `line-length = 88` - can this be consolidated?

Reply:
I put this to keep consistency with the flake8 rules, which we had set to [120 before](https://github.com/basetenlabs/truss/blob/b8fcdc390b18b8b57cc347467053c06d2c7ab3a3/.flake8), where black was still enforcing 88 (the default) for formatting purposes. 

If we want to move up the max line line length for formatting though, happy to do that. Might make sense in another PR to avoid an overly large diff though.

Reply:
Oh I was thinking more about clipping `120 -> 88`. Is the effect of having a longer line limit here to allow some wiggle space, like URLs in docstrings?

In that case that makes sense to keep (but add a comment with that rationale).

Reply:
Exactly! Let me add a comment on this.

Reply:
Since the formatter technically can go above the line length, so it's good to have this be a bit higher.